examples/vmod_counters/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+use varnish::vsc_wrapper::Vsc;
+use varnish::Stats;
+
+#[derive(Stats)]
+#[repr(C)] // required for correct memory layout
+pub struct VariousStats {
+    /// Number of hits
+    #[counter]
+    hits: std::sync::atomic::AtomicU64,
```

Comment:
```suggestion
    hits: AtomicU64,
```

and add `use std::sync::atomic::AtomicU64` at the top